scripts/index.js

```diff
@@ -0,0 +1,44 @@
+var blog =
+$('article:first').clone()[0];
+
+article.prototype.toHTML = function (argument) {
+  $artSkel = $('article').filter(':first').clone();
+};
+
+function postArticle (rawArticle) {
+  var art = new ARticle(rawArticle);
+  art.toHTML();
+};
+
+var Article = function (props) {
+  this.author = props.authoer;
+  this.whatever = props.whatever;
+  articles.push(this);
+};
+
+Article.prototype.toHTML = function() {
+  var $clonedARticle = $('article#template').clone();
+  $clonedArticle.removeAttr('id');
+  $clonedArticle.find('h1.author').html('<a href "' + articles[i].authorURL + '" ' + articles[i].author);
```

Comment:
Gotcha. Just to keep your master branch clean, you may want to not commit this to a pull request to master if it's not being used unless there is good reason to include it.